api/services/S3SiteRemover.js

```diff
@@ -41,11 +52,9 @@ const getKeys = (s3Client, prefix) => s3Client.listObjects(prefix)
 const removeInfrastructure = (site) => {
   if (site.s3ServiceName !== config.s3.serviceName && site.awsBucketName !== config.s3.bucket) {
     return apiClient.deleteRoute(site.awsBucketName)
+      .catch(handleError)
```

Comment:
i did notice that and wanted to ask @apburnes  if there was a reson for not having .then(() =>

we need both catches because we if if the deleting Route throws an error because the route doesn't exist we want to delete the serviceInstance and if the service instance doesn't exist the site can still be deleted

Reply:
> Not sure why you would need both `catch`es here, one at the end would catch errors from both parts. Maybe you are observing some issues because I think there is an error on the following line, maybe you want
> 
> ```
> .then(() => apiClient.deleteServiceInstance(site.s3ServiceName))
> ```
> 
> instead of
> 
> ```
> .then(apiClient.deleteServiceInstance(site.s3ServiceName))
> ```
> 
> ?

addressed